AnacondaRecipes/mccabe-feedstock
# mccabe 0.7.0

Update mccabe to 0.7.0

Version change: bump version number from 0.6.1 to 0.7.0
Bug Tracker: new open issues https://github.com/PyCQA/mccabe/issues
Github releases:  https://github.com/PyCQA/mccabe/compare/0.6.1...0.7.0
Upstream license:  License file:  https://github.com/PyCQA/mccabe/blob/master/LICENSE
Upstream setup.py:  https://github.com/PyCQA/mccabe/blob/0.7.0/setup.py

The package mccabe is mentioned inside the packages:
flake8 | pylint | python-language-server | python-lsp-server | radon |

Versions used by other packages
```
'>=0.6.0,<0.7.0':
- flake8 3.9.2
- python-language-server 0.36.1
- python-lsp-server 1.2.4
'>=0.6,<0.7':
- pylint 2.9.6
```

Actions:
1. Reset build number to `0`
2. Use `noarch: python`
3. Update script
4. Add `pip` and `wheel` in `host`
5. Fix python in `run` and `test/requires`
6. Add `pip check`
7. Add doc_url

Result:
- all-succeeded
